packages/sdks/output/angular/scripts/multi-build.mjs

```diff
@@ -0,0 +1,103 @@
+import babel from '@babel/core';
+import tsPlugin from '@babel/plugin-syntax-typescript';
+import { execSync } from 'child_process';
+import fs from 'fs';
+import glob from 'glob';
+import path from 'path';
+import prettier from 'prettier';
+
+/**
+ * @typedef {import('@babel/core').PluginObj} babel.PluginObj
+ * @typedef {babel.PluginObj} InlinePlugin
+ */
+
+const sdkEnv = process.env.SDK_ENV;
+
+if (!sdkEnv) {
+  throw new Error('SDK_ENV is required to build the SDK.');
+}
+
+const ngPackageJsonPath = path.resolve('ng-package.json');
+const ngPackageJson = JSON.parse(fs.readFileSync(ngPackageJsonPath, 'utf-8'));
+
+const outputPath = {
+  node: './lib/node',
+  browser: './lib/browser',
+}[sdkEnv];
+
+if (!outputPath) {
+  throw new Error(
+    `Unknown SDK_ENV: ${sdkEnv}. Expected one of 'node', 'browser'.`
+  );
+}
+
+// Set the destination path to the correct output path for specific SDK_ENV
+ngPackageJson.dest = outputPath;
+
+fs.writeFileSync(
+  ngPackageJsonPath,
+  // we needed to prettify the JSON to make sure it matches the linting rules
+  await prettier.format(JSON.stringify(ngPackageJson, null, 2), {
+    parser: 'json',
+  }),
+  'utf-8'
+);
+
+const foundFiles = glob.sync('src/**/*/choose-eval.*');
+
+if (foundFiles.length !== 1) {
+  throw new Error(
+    'Expected exactly one file named `choose-eval.ts` containing "placeholder-runtime" import.'
+  );
+}
+
+const chooseEvalFile = foundFiles[0];
+
+const getFolderName = () => {
+  return sdkEnv === 'node' ? 'node-runtime' : 'browser-runtime';
+};
+
+// Helper function to transform the file
+const transformFile = (filePath, replaceValue, revert = false) => {
+  const result = babel.transformFileSync(filePath, {
+    plugins: [
+      tsPlugin,
+      /** @returns {InlinePlugin} */
+      () => {
+        return {
+          visitor: {
+            ImportDeclaration(path) {
+              const currentValue = revert
+                ? replaceValue
+                : 'placeholder-runtime';
+              const newValue = revert ? 'placeholder-runtime' : replaceValue;
+              if (path.node.source.value === currentValue) {
+                path.node.source.value = newValue;
+              }
+            },
+          },
+        };
+      },
+    ],
+  });
+
+  fs.writeFileSync(filePath, result.code, 'utf-8');
+};
+
+// Transform the choose-eval.ts file to replace 'placeholder-runtime'
+// to appropriate runtime value based on the SDK_ENV
+const folderName = `./${getFolderName()}/index`;
```

Comment:
you should be able to reuse the logic from `@builder.io/sdks/output-generation`. check how we do that in the other SDKs:

![CleanShot 2024-07-11 at 09 37 41@2x](https://github.com/BuilderIO/builder/assets/1393142/8566b266-1fd1-4c8d-bfc7-edf6778d359d)

this should remove a bunch of logic from this file

Reply:
thank you, yes I've now re-used most of the functions from this file

Reply:
You should be able to use `getEvaluatorPathAlias` here

```suggestion
// Transform the choose-eval.ts file to replace 'placeholder-runtime'
// to appropriate runtime value based on the SDK_ENV
const folderName = getEvaluatorPathAlias({ format: 'js' });
```

Reply:
done! in angular we cannot directly use an extension so I've made a minor update to `getEvaluatorPathAlias` so that if a `format` is mentioned then only use it else don't add the extension. I have also updated `getEvaluatorPathAlias` in places where it was using the default format (ts) -- only one